src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -45,6 +49,15 @@ public void Configure(TelemetryConfiguration configuration)
                 configuration.InstrumentationKey = this.applicationInsightsServiceOptions.InstrumentationKey;
             }
 
+            if (this.telemetryProcessorFactories.Any())
```

Comment:
I think whoever adds manually to the `TelemetryProcessorChainBuilder` should also take care of calling `Build()`. We do, so we call. Calling `Build()` "just in case" is a little dubious. The original code did not do that and I did not want to change this.